src/main/scad/laszlo-molnar/l-i-vator-ii/molnar.l-i-vator-ii.scad

```diff
@@ -0,0 +1,82 @@
+include <puzzlecad.scad>
+
+box_puzzle_border = 6;
+
+$burr_inset = 0.125;
+$burr_bevel = 1;
+$unit_beveled = true;
+$burr_scale = 17;
+
+dim = $burr_scale * 3 + box_puzzle_border * 2 + $burr_inset * 2;
+height = $burr_scale * 3 + box_puzzle_border + $burr_inset * 2;
+
+mid = box_puzzle_border / 2 + 0.5;
+far = dim - mid;
+    
+*box();
+*pieces();
+*cap();
+
+module box() {
+    render(convexity = 2)
+    difference() {
+        beveled_cube([dim, dim, height]);
+        translate([box_puzzle_border, box_puzzle_border, box_puzzle_border])
+        cube([
+            $burr_scale * 3 + $burr_inset * 2,
+            $burr_scale * 3 + $burr_inset * 2,
+            $burr_scale * 4
+        ]);
+     }
+     
+     translate([0, 0, height])
+     connector(1.8, 2.0);
+}
+
+module pieces() {
+    burr_plate([
+        ["xx|xx", ".x|x.", "..|x."],
+        ["x.|xx", "..|x.", "..|x."],
+        ["xx|xx", "..|x.", "..|x."],
+        ["x.|xx", "..|x."],
+        [".x|xx"],
+        ["xx"]
+    ], $plate_width = 160);
+}
+
+module cap() {
+    render(convexity = 2)
+    difference() {
+        beveled_cube([dim, dim, box_puzzle_border]);
+        translate([box_puzzle_border, box_puzzle_border, 
+        -0.01
```

Comment:
maybe make this -0.001

a difference of 0.01 is actually enough to affect the fit (at least on my very accurate Prusa!)